src/iptraf.c

```diff
@@ -252,6 +252,7 @@ static void program_interface(void)
 static const char *const iptraf_ng_usage[] = {
 	IPTRAF_NAME " [options]",
 	IPTRAF_NAME " [options] -B [-i <iface> | -d <iface> | -s <iface> | -z <iface> | -l <iface> | -g]",
+// PHIL need to add: -t timeout min | -L filename | -f | -h
```

Comment:
changes not related to docs, leave it to another changeset

Reply:
deleted line